### PR TITLE
NTBS-2261: Update forest extract population to use NTBS notification as base, not reusable notification

### PR DESCRIPTION
**Pull request by LeoCie**

### CODE REVIEW

File: source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql [hunk 1]
```diff
@@ -69,71 +69,85 @@ BEGIN
...
 			'Positive'																						AS SmearSampleResult,
 			lab.SpecimenDate																				AS SpecimenDate,
 			lab.ReferenceLaboratoryNumber																	AS ReferenceLaboratoryNumber,
 			GETUTCDATE()																					AS ExtractDate,
 			patient.LocalPatientId																			AS LocalPatientId,
-			Age																								AS Age,
-			--hospital.CaseManagerUsername																	AS OwnerUserId,
-			CaseManager																						AS CaseManager,
-			rn.Consultant																					AS PatientsConsultant,
+			(0 + Convert(Char(8),GETDATE(),112) - Convert(Char(8),patient.Dob,112)) / 10000					AS Age,
```
Comment (JoshCadney) [inline source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql:120]: What is happening on this line?

File: source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql [hunk 2]
```diff
@@ -69,71 +69,85 @@ BEGIN
...
+			--hospitalDetails.CaseManagerUsername															AS OwnerUserId,
```
Comment (JoshCadney) [inline source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql:121]: I think this comment can be removed

**Review by JoshCadney — COMMENTED** [review]
Looks good - but one question

File: source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql (hunk 1 above)
Comment (aptemus) [reply source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql:120]: I think that's a very complicated way to work out the patient's age at time of notification - we have a function that does that, which can be called by passing in date of birth and notification date - https://github.com/publichealthengland/ntbs-reporting/blob/master/source/dbo/Functions/Reusable%20Notification/ufnGetAgefrom.sql

Comment (LeoCie) [reply source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql]: Using the function now, I thought we might have one but forgot to look

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

**Pull request merged**